Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -305,6 +334,41 @@ def runSelected(opt):
                       default=None,
                       action='store')
 
+    parser.add_option('--RequiresGPU',
+                      help='if GPU is reuired or not: forbidden (default, CPU-only), optional, required. For relvals, the GPU option will be turned off for optional.',
+                      dest='RequiresGPU',
+                      default='forbidden')
+
+    parser.add_option('--GPUMemoryMB',
+                      help='to specify GPU memory. Default = 8000 MB (for RequiresGPU = required).',
+                      dest='GPUMemoryMB',
+                      default='8000')
+
+    parser.add_option('--CUDACapabilities',
+                      help='to specify CUDA capabilities. Default = 7.5 (for RequiresGPU = required). Use comma to identify various CUDACapabilities',
```

Comment:
This is trickier to set automatically.
Also, I think the latest changes to the CUDA build rules mean that we support anything like `6.0` or above.

Reply:
Fixed. The help was wrong. The default value starts from 6.0.